3_visualize/src/drought_learner_viz_components.R

```diff
@@ -885,5 +891,153 @@ frame_l <- function(p2_streamflow_learner_viz_df, p2_droughts_learner_viz_df,
   ggsave(out_png, width = 1200, height = 800, dpi = 300, units = "px")
 }
 
-
+frame_m <- function(p2_streamflow_learner_viz_df, p2_droughts_learner_viz_df,
+                    p2_1951_2020_drought_prop_site, p2_1951_2020_drought_prop_jd_7d,
+                    bottom_bars, canvas, inset, out_png){
+  
+  # Select correct station ID and threshold
+  p2_droughts_site <- p2_1951_2020_drought_prop_site %>%
+    filter(StaID == '03221000', threshold == 10) %>%
```

Comment:
SC: When you do your refactoring, I'd either a) provide the station id and threshold value as arguments (see my note below re:  modularizing and separating the data prep steps) or b) define them as targets in `2_process.R` , in which case they could be referenced to build `p2_droughts_learner_viz_df` and the other dataframes for the learner viz, and when prepping the data for this plot.

Since this pipeline is so customized to one site, and that is unlikely to change, I don't think b) is necessary, but I do think it would be good for a user to be able to see in the target recipe files which site and which threshold is being used for the plots, whether that's as function arguments (a) or standalone targets (b). As is, that information (for this plot) is buried in this function.